vllm/entrypoints/openai/protocol.py

```diff
@@ -61,6 +61,7 @@ class ChatCompletionRequest(BaseModel):
     max_tokens: Optional[int] = None
     stop: Optional[Union[str, List[str]]] = Field(default_factory=list)
     stream: Optional[bool] = False
+    logprobs: Optional[int] = None
```

Comment:
Could you change the type to bool, which aligns with openAI spec. Also I think in the current codebase, it uses it as bool as well. Please correct me if I'm wrong.

Could you change the type definition of logprobs in CompletionRequest as well? thanks.